doc/classes/PopupMenu.xml

```diff
@@ -636,6 +642,7 @@
 		</member>
 		<member name="prefer_native_menu" type="bool" setter="set_prefer_native_menu" getter="is_prefer_native_menu" default="false">
 			If [code]true[/code], [MenuBar] will use native menu when supported.
+			[b]Note:[/b] If [PopupMenu] is linked to [StatusIndicator], [MenuBar] or another [PopupMenu] item it can use native menu regardless of this property, use [method is_native_menu] to check it.
```

Comment:
```suggestion
			[b]Note:[/b] If [PopupMenu] is linked to [StatusIndicator], [MenuBar], or another [PopupMenu] item it can use native menu regardless of this property, use [method is_native_menu] to check it.
```